hash/compis.xml

```diff
@@ -216,12 +216,118 @@ Systemskiva för hårdskiveenhet [version 7242]
+		<info name="serial" value="24-33584" />
+		<info name="release" value="19870114" />
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="2209365">
+				<rom name="Action1 Glosprogram.mfm" size="2209365" crc="1e2a96a1" sha1="098842f0571c0b9beb64295bef5ded29e5ce6add"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="coulomb">
+		<description>Coulombs lag (nät/skollicens)</description>
+		<year>1987</year>
+		<publisher>Esselte Studium</publisher>
+		<info name="serial" value="24-33772" />
+		<info name="release" value="19870731" />
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="2209453">
+				<rom name="Coulombs lag.mfm" size="2209453" crc="16e9749c" sha1="adc4658eaa24cdaf4f261f2a1848b98435155d0a"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="kinetik">
+		<description>Kinetik (nät/skollicens)</description>
+		<year>1987</year>
+		<publisher>Esselte Studium</publisher>
+		<info name="serial" value="24-34652" />
+		<info name="release" value="19870823" />
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="2209459">
+				<rom name="Kinetik.mfm" size="2209459" crc="c906fa1d" sha1="a6e9411dcc1935cb17a65c7eb162fc127eb1f9a1"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="jordskift">
+		<description>Jorden skiftas, folket skingras (nät/skollicens)</description>
+		<year>1987</year>
+		<publisher>Esselte Studium</publisher>
+		<info name="serial" value="24-33799" />
+		<info name="release" value="19871105" />
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="2209529">
+				<rom name="Jorden skiftas, folket skingras.mfm" size="2209529" crc="b1df76ea" sha1="6d6e2fd20ccc4cec2999932168f8a82221f16c47"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<!-- Requires a hardware key which isn't dumped -->
+	<software name="skvallra">
+		<description>Får dataregister skvallra? (enanvändare)</description>
+		<year>1985</year>
+		<publisher>Esselte Studium</publisher>
+		<info name="serial" value="24-34112" />
+		<info name="release" value="19851029" />
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="2209468">
+				<rom name="Får dataregister skvallra.mfm" size="2209468" crc="06fc17bc" sha1="7de8a56025058f2a5298ee0dfa1db32c7f335643"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="skvallrak" cloneof="skvallra">
+		<description>Får dataregister skvallra? (komplement)</description>
+		<year>1986</year>
+		<publisher>Esselte Studium</publisher>
+		<info name="serial" value="24-34820" />
+		<info name="release" value="19860904" />
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="2152177">
+				<rom name="Får dataregister skvallra (Komplement).mfm" size="2152177" crc="9fa1f31a" sha1="8e0eb4e26d4167193f2b4504f014cf14a8ab8c65"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<!-- Requires a hardware key which isn't dumped -->
```

Comment:
I've been wondering: does this software which requires an undumped hardware key work in MAME? Otherwise it should be marked as supported="no"

Reply:
I'm not sure any software works in mame.